spec/jobs/signal_adapter/receive_polling_job_spec.rb

```diff
@@ -97,6 +98,12 @@
           subject.call
           expect(contributor.reload.signal_onboarding_completed_at).to be_present
         end
+
+        it 'enqueues a job to attach contributors avatar' do
+          expect(SignalAdapter::AttachContributorsAvatar).to receive(:perform_later).with(contributor)
+
+          subject.call
+        end
```

Comment:
I think instead of this you can simply do:

```suggestion
        it { should have_enqueued_job(SignalAdapter::AttachContributorsAvatar).with(contributor) }
```

https://relishapp.com/rspec/rspec-rails/docs/matchers/have-enqueued-job-matcher